### DISCUSSION TITLE
Customizable options for checkboxes

### DISCUSSION THREAD

**Discussion by hatredholder** [damionx7/Zettel-Notes-Documentation#176] [Feature Requests]
**Feature Request**: customizable options for **checkboxes**/**tasks**

**Use Case**: marking task progress easily

I have been testing the app with my lists and encountered a minor annoyance in my workflow. I often use symbols other than "X" in my checkmarks to indicate various statuses such as progress, cancellation, rescheduling, etc.

I was wondering if an option could be added to support a couple of other "custom" symbols for the checkboxes.

Here's an example of how it would look like:
![342312571-1d457c0b-9718-4dfd-a0e6-7e84fc2da528](https://github.com/user-attachments/assets/969b1c2c-c9a7-4358-8441-d7df5219f32c)

**Comment by damionx7** [comment]
Can you suggest any applications that use this for tasks ? I would like to try this feature in those apps.

**Reply by hatredholder** [reply]
I got the idea from Obsidian. Though, to have those custom checkboxes you must install some kind of theme, the [Minimal](https://github.com/kepano/obsidian-minimal) theme is what I used and [this](https://github.com/kepano/obsidian-minimal?tab=readme-ov-file#alternate-checkboxes) is exactly what you're looking for.

**Reply by damionx7** [reply]
Thanks. So, all these checkboxes mean the task is checked.

**Reply by hatredholder** [reply]
I guess, yeah
